dataclients/kubernetes/ingress.go

```diff
@@ -399,7 +399,7 @@ func (ing *ingress) convert(state *clusterState, df defaultFilters, r *certregis
 			if err != nil {
 				return nil, err
 			}
-			if r != nil {
+			if r != nil && r.UniqueRoute(routes) {
```

Comment:
Both `LoadAll` and `LoadUpdate` use `mapRoutes(r) `:
https://github.com/zalando/skipper/blob/504a5e05e301bb1f19c7e6a1a07325d24d4a763f/dataclients/kubernetes/kube.go#L458-L484

Lets check if we can utilize it to detect duplicates at one place.